hedera-node/hedera-app/src/test/resources/bootstrap.properties

```diff
@@ -78,8 +78,8 @@ contracts.keys.legacyActivations=1058134by[1062784]
 contracts.knownBlockHash=
 contracts.localCall.estRetBytes=32
 contracts.maxGasPerSec=15000000
-contracts.maxKvPairs.aggregate=500_000_000
-contracts.maxKvPairs.individual=163_840
+contracts.maxKvPairs.aggregate=500000000
```

Comment:
I'm not sure why the underscores were removed. They are helpful when verifying the correct number is set.

Reply:
The underscores were causing some of the HAPI tests to fail in the modularized code with `NumberFormatException`.  Apparently, the logic for parsing the config values with the underscores has not been implemented?  We probably went overboard on removing them but we wanted to be sure that the tests would pass.

Reply:
@lukelee-sl can we open a ticket to fix that in modularization. 
FYI @hendrikebbers

Reply:
looks like an issue was already created by @david-bakin-sl 
https://github.com/hashgraph/hedera-services/issues/9589